### DISCUSSION TITLE
How can I prevent Slowloris attacks in quarkus?

### DISCUSSION THREAD

**Discussion by bhushanshinde07** [quarkusio/quarkus#23130] [Q&A]
I am trying to protect my quarkus application from Slowloris attacks so Is there any configuration setting or way available in the quarkus?

**Comment by sberyozkin** [comment]
Do you mean you'd like to control a number of possible connections ? Please see https://stackoverflow.com/questions/64493361/how-to-setting-http-thread-pool-max-size-on-quarkus. This kind of guard may better be controlled by the gate in front of Quarkus
